### ISSUE TITLE
Ratings

### ISSUE THREAD

**Issue by mkozhukharenko** [ifnul/ums-backend#18]
- [x] Додати фікстур для `enrolments/${specofferId}/ratings`
  
  ``` json
  {  "firstname" : "a1", "fathername" : "f1", "surname" : "a1" }
  // =>>
  {   "firstname" : "sadfa",  "fathername" : "afds",  "surname" : "fsddfds" }
  ```
- [x] Округлити поле kb. Наразі трапляються такі значення як `228.07999999999998`
- [ ] ~~Додати поле `name` для enrolmnet та specoffer~~  перенести до іншого таску
- [x] Потрібен ідентифікатор прохідного балу (щоб відобразити "червону межу" вище якої - прохідний бал")

**Comment by ivanursul** [comment]
1 вже є 
![image](https://cloud.githubusercontent.com/assets/3056498/14232547/4ae4fc82-f9b6-11e5-8026-65c606081f9d.png)

**Comment by ivanursul** [comment]
з приводу name - що це має бути ? Бо ніби ж є firstname lastname

**Comment by ivanursul** [comment]
Ідентифікатор прохідного балу додам

**Comment by ivanursul** [comment]
Добавив ідентифікатор, трошки помінявся респонс

**Comment by mkozhukharenko** [comment]
> з приводу name - що це має бути ? Бо ніби ж є firstname lastname

Ти написа  в скайпі:

> Ще треба буде добавити name в enrolments i specoffer

Здається ти мав на увазі інші api-шки просто.

> Добавив ідентифікатор, трошки помінявся респонс 

поліз в скайп з;ясовувати чому зламався формат responce-у а відповідь знайшов тут  :)

**Comment by mkozhukharenko** [comment]
Як на FE використовувати ці данні? 

``` js
licCount:100
stateCount:20
```

**Comment by ivanursul** [comment]
stateCount - ніби кількість державних місць

**Comment by mkozhukharenko** [comment]
зрозуміло. Це потрібно якимось чином відобразити на UI-ці?

**Comment by ivanursul** [comment]
Угу, як ми говорили. чи якоюсь лінією, чи як

**Comment by mkozhukharenko** [comment]
Тобто лише перших 20 row-ів, як я зрозумві (в данному випадку)
licCount - тоді непотрібний взагалі?

З сервверу вже приходять посортовані enrolmen-ти, вірно?

**Comment by ivanursul** [comment]
так, licCount - це недержавні місця, треба уточнити в Іллі
Так, з серверу приходять посортовані

**Comment by mkozhukharenko** [comment]
stateCount  - це і є  ідентифікатор прохідного балу ?

**Comment by ivanursul** [comment]
Так

**Comment by mkozhukharenko** [comment]
ок. Я додав "червону межу" ( якщо 120 заявок і stateCount= 20 то червона межа буде після 20ї заявки)

**Comment by mkozhukharenko** [comment]
можливо і licCount потрібно якось відобазити на UI? якщо licCount=100 то якусь межу провести після 100ї заявки?

**Comment by ivanursul** [comment]
Та, можна добавити і межу для liccount. А можливо відобразити якось зверху опис що означає червоний і синій колір ?

**Comment by mkozhukharenko** [comment]
так, думаю що було б корисно. А який текст(опис) має бути до цих ліній? (щоб не написати якусь нісенітницю)

**Comment by mkozhukharenko** [comment]
ось так поки зробив
![image](https://cloud.githubusercontent.com/assets/9365923/14721223/063065fc-080e-11e6-9e41-55f87dd50475.png)
